OneSignalSDK/unittest/src/test/java/com/onesignal/ShadowOneSignalRestClient.java

```diff
@@ -75,25 +75,44 @@
       }
    }
 
-   public static JSONObject lastPost;
+   public static List<String> successfulGETResponses = new ArrayList<>();
    public static ArrayList<Request> requests;
-   public static String lastUrl;
+   public static String lastUrl, failResponse, nextSuccessResponse, nextSuccessfulGETResponse, nextSuccessfulRegistrationResponse, pushUserId, emailUserId, failMethod;
+   public static Pattern nextSuccessfulGETResponsePattern;
+   public static JSONObject lastPost;
    public static boolean failNext, failNextPut, failAll, failPosts, failGetParams;
    public static int failHttpCode;
-   public static String failResponse, nextSuccessResponse, nextSuccessfulGETResponse, nextSuccessfulRegistrationResponse;
-   public static Pattern nextSuccessfulGETResponsePattern;
-   public static List<String> successfulGETResponses = new ArrayList<>();
    public static int networkCallCount;
 
-   public static String pushUserId, emailUserId;
-
-   public static JSONObject paramExtras;
    
    // Pauses any network callbacks from firing.
    // Also blocks any sync network calls.
    public static boolean freezeResponses;
    private static ConcurrentHashMap<Object, PendingResponse> pendingResponses = new ConcurrentHashMap<>();
 
+   private static String remoteParamsGetHtmlResponse = null;
+   private static final String REMOTE_PARAMS = " {" +
```

Comment:
Should we construct a `JSONObject` and use `toString` method instead of hardcoding this?

Reply:
Its a mocked hardcoded response for testing, from my point of view, is easier to read in this way since there is a lot of data. But if you prefer to have it as a JsonObject yeah that wouldn't harm, let me know

Reply:
👍